templates/asp-app.json

```diff
@@ -153,6 +153,9 @@
         "environment": "[parameters('env')]",
         "Team Name": "[parameters('teamName')]"
       },
+      "dependsOn": [
```

Comment:
this change should be fine to be mainlined if you separate it out to a different PR